### PR TITLE
doc: run install.bat on libvirt

### PR DESCRIPTION
**Pull request by oskardotglobal**
This PR does two things:
1. Document running install.bat on libvirt
2. Apply the fix from #279 only if using Podman/Docker

### CODE REVIEW

**Comment by oskardotglobal** [comment]
Can someone review?

File: README.md [hunk 1]
```diff
@@ -282,7 +282,7 @@ The following guides are available:
 - [Creating a Windows VM with `Docker` or `Podman`](docs/docker.md)
 - [Creating a Windows VM with `libvirt`](docs/libvirt.md)
 
-If you already have a Windows VM or server you wish to use with WinApps, you will need to merge `install/RDPApps.reg` into the Windows Registry manually.
+If you already have a Windows VM or server you wish to use with WinApps, you will still have to still follow the [final steps described in the `libvirt` documentation](https://github.com/winapps-org/winapps/blob/main/docs/libvirt.md#final-configuration-steps).
```
Comment (LDprg) [inline README.md:285]: This seems like a bit of still too much

File: bin/winapps [hunk 2]
```diff
@@ -4,6 +4,8 @@
 # Copyright (c) 2024 Rohan Barar
 # Copyright (c) 2024 Oskar Manhart
 # Copyright (c) 2024 escapefreeg
+# Copyright (c) 2024 pre-commit-ci[bot]
```
Comment (LDprg) [inline bin/winapps:7]: There are less copy right notice in this file, seems a bit odd to me

**Review by LDprg — COMMENTED** [review]
Seems good, but the Copyright notice seems a bit odd?

File: bin/winapps (hunk 2 above)
Comment (oskardotglobal) [reply bin/winapps:7]: Honestly, lictool is just such a cursed piece of software
I don't even know why it didn't list @actions here

File: apps/acrobat9/info [hunk 3]
```diff
@@ -1,5 +1,8 @@
 # Copyright (c) 2024 Adam
 # Copyright (c) 2024 Oskar Manhart
+# Copyright (c) 2024 pre-commit-ci[bot]
+# Copyright (c) 2024 github-actions[bot]
```
[the remark this answers is not in the document]
Comment (oskardotglobal) [reply apps/acrobat9/info:4]: No, this is definitely not right
I have no idea why it does that ....

**Comment by oskardotglobal** [comment]
Sorry for this absolute mess of a PR but I've cleaned everything up now

**Review by LDprg — APPROVED** [review]
Seems good

**Comment by AkechiShiro** [comment]
I'm sorry I dont have time to review this on my end.

**Pull request merged**